migrations/V208__Update_MTO_Common_Solution_Contact.sql

```diff
@@ -56,5 +56,32 @@ CHECK (
     (mailbox_address IS NULL AND mailbox_title IS NULL AND user_id IS NOT NULL AND is_team = FALSE)
 );
 
--- This needs to be part of a seperate migration than the definition of contractor table to avoid issues with existing data 
+DROP INDEX idx_unique_primary_contact_per_mto_common_solution;
```

Comment:
Thanks @Peyton232, something to note here, there is nothing in these constraints that requires a primary contact. It only will error if there are two primary contacts. This should probably be solved for before merging the feature to main.